frontend/src/components/ExternalDb.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <section class="section extended-section">
+    <div v-if="externalDb" class="container is-fullhd">
+      <h3 class="title">
```

Comment:
Last time I checked, this is what the Bulma guideline, also matching the rest of our codebase:

```suggestion
      <h3 class="title is-3">
```

Reply:
Can you point me to the guideline please? I did this since `is-3` is the default for `is-title` (see https://bulma.io/documentation/elements/title/#sizes). If it's a part of the guideline then I'm all for it but if not I would actually want to open up a new issue to remove it from the rest of the codebase.

Reply:
It was right there at the link you shared:
<img width="335" alt="image" src="https://user-images.githubusercontent.com/23480589/119793164-a24a7700-bed6-11eb-9043-26c7970b9027.png">
Since it's the default value I guess we could skip it, but for clarity and consistency I vote for having it.
Also, I meant that at the moment this is how things are in the rest of the codebase - a heading tag with class `title` also has the size specified. I might be wrong, but I have the memory of having done this check at some point, so there should be no need of opening an issue.

Reply:
Ah I thought you meant specifically that there was documentation that mentions you should always include the default modifiers. I feel like it's good practice to rely on the framework and avoid default modifiers when possible but it's not a strong preference. So we can merge this change and not open a new issue.

@inghylt might be working on this branch atm so I will leave it for her to merge this suggestion to avoid potential branch conflicts.